pages/linux/systemctl.md

```diff
@@ -31,6 +27,10 @@
 
 `systemctl daemon-reload`
 
-- Check if a unit is enabled:
+- Check if a unit is active/enabled/failed:
```

Comment:
typo: "an unit"

Reply:
No, "a unit" is correct. The usage of "a" or "an" depends on the phoneme of the next word, not the spelling. The "u" in unit is pronounced as a "y", hence "a" instead of "an".

Reply:
English is confusinsg and frankly stupid sometimes, I know.